pkgs/servers/matrix-synapse/0002-Expose-generic-worker-as-binary-under-NixOS.patch

```diff
@@ -0,0 +1,43 @@
+From 3089758015c64cc1e6788793c4fe40a0e1783457 Mon Sep 17 00:00:00 2001
+From: Maximilian Bosch <maximilian@mbosch.me>
+Date: Tue, 5 Oct 2021 22:33:12 +0200
+Subject: [PATCH 2/2] Expose generic worker as binary under NixOS
```

Comment:
Is there a reason we haven't upstreamed this? That looks like something that wouldn't hurt upstream at all.

Reply:
Let's see: https://github.com/matrix-org/synapse/pull/11449